app/webpack.common.js

```diff
@@ -57,8 +57,13 @@ const replacements = {
 
 const outputDir = 'out'
 
+let externals = ['7zip']
+if (replacements.__DEV__) {
```

Comment:
I'd rather we checked `environment === 'development'` here than checking our replacements.